src/bin/pgcopydb/schema.c

```diff
@@ -1861,72 +1785,6 @@ struct FilteringQueries listSourceTablesNoPKSQL[] = {
 	}
 };
 
-/*
- * schema_list_ordinary_tables_without_pk lists all tables that do not have a
- * primary key. This is useful to prepare a migration when some kind of change
- * data capture technique is considered.
- */
```

Comment:
Why removing that? I think we could discuss about removing `pgcopydb list tables --without-pkey` in a separate PR, but even then, I'm not sure why we would remove the feature. Seems useful to me.